prime-router/src/testIntegration/resources/settings/organizations.yml

```diff
@@ -172,7 +171,7 @@
       organizationName: "elims"
       format: "HL7"
       customerStatus: "testing"
-      schemaName: "metadata/fhir_transforms/senders/default-sender-transform"
+      schemaName: "metadata/fhir_transforms/senders/original-pipeline-transforms"
```

Comment:
I'm not sure we want elims to be using the original pipeline transforms? I thought the point was to remove the dependency of ELIMS on the default sender transform? With the changes as is, all you're doing essentially is just removing the contents of elims-sender-transform from the transform that ultimately gets applied to their messages?

Reply:
Odd it was removed at one point. Removed again.